### ISSUE TITLE
Stereo version request

### ISSUE THREAD

**Issue by vcvrackideas** [suzumushi0/SoundObject_binary#3]
Hi! Your software is awesome. Would be stereo version ever possible? Now I'm using custom 2 way configuration and it sound really great for virtualization of stereo speakers in the near field.

**Comment by suzumushi0** [comment]
Could you clarify what the stereo version is? 

The latest version already supports it if it means monaural signal input and stereo speakers output.

If it inputs stereo speakers signal then outputs binaural headphones signal, that is, it emulates stereo speakers-based sound field by headphones, it could be implemented by the following procedure.

(1) prepare 3 tracks for stereo signal, left channel signal, and right channel signal.
(2) stop the master send of the stereo track.
(3) send the left signal of the stereo track to the left track.
(4) send the right signal of the stereo track to the right track.
(5) insert SoundObject to left and right tracks.
(6) set acoustic positions as speaker positions.
(7) mix outputs of SoundObject.
![pic](https://user-images.githubusercontent.com/67182469/153537637-1d319732-2717-4a61-bcf1-db5e8e5c442e.png)

**Comment by vcvrackideas** [comment]
Yes, I did that kind of connection yesterday. It works nicely except two things - first can be done by entering similar value to achieve same copy to opposite object instance. But the second one it though - there is no "reflection" when panning signal on the opposite side, like in the real room...and I can't achieve that, only with crossfeed after them but it can kill overal perception I guess

**Comment by suzumushi0** [comment]
I can't understand the second issue. Could you explain in clearer English?

**Comment by vcvrackideas** [comment]
> I can't understand the second issue. Could you explain in clearer English?

Sorry, I used Ableton and  got confused with full panning of left and right tracks
Now I understand that they both don't need to be panned, but just mixed after sound object processing. I'm going to use it systemwide on windows for all system sound and I think I need some capturing of true stereo IR to achieve that DAW configuration in simple vst host then. Thanks for proper explanation!

**Comment by vcvrackideas** [comment]
Just for the record - I spent more than year in attempts of finding perfect solution for headphones externalization and seems I found it in your plugin. That high resolution Kemar and reflected waves output give a really nice sense of my studio room where finally that headphones side bass muddy effect is gone together with overall sense of compression and transients now are so revealing as well. Now I'm going to replicate measurements of my room and my listening position correctly and try it again.

**Comment by suzumushi0** [comment]
Thank you so much for your invaluable feedback.

(1) Yes, outputs of SoundObject should be simply mixed. It is clear from the superposition principle.

(2) As I described above message, a stereo speakers-based acoustic field could be emulated by SoundObject. I think implementing a dedicated version for that purpose depends on demand.

(3) I am glad to hear that Aachen Kemar IR is nice for you. Frankly, it is not nice for me :-). It may depend on personal. Anyway, it is very informative feedback.

Thanks again, if you have further questions or feedback, please get in touch with me.

**Comment by vcvrackideas** [comment]
Very nice about possible dedicated stereo sound field version. Because all commercial products just work wrong at least for me.
Noticed some bugs when loaded through vst3 wrapper such as Blue cat patchwork. If you have chance to grab their demo you will get what I mean - no blue rectangle of the room and manual entered coordinates just don't respond correctly.
Yesterday I tried captured an IR (true stereo) with ableton dedicated tool to apply on my systemwide convo reverb. It soundd the same as two Objects instances but I have noticed some phasing effect in the pure center of stereo field. I should compare it with speakers more or possibly switch to open headphones instead close but I'd like to hear your opinion on captured Object IR like that. According the manual the plugin constantly adds doepler effect if I understood that correctly but probably I'm wrong and it applies only if the source position is in the moving state.

**Comment by suzumushi0** [comment]
You are correct. SoundObject does not add Doppler effect when an acoustic source is not moving.

I'm not sure what you are trying now but I would like to suggest locating the center of the sphere to an asymmetric position in the reverberation room. This is because if it is symmetric, large peaks and dips are observed in frequency characteristics.

**Comment by vcvrackideas** [comment]
I'm trying to replicate my mixing room to fully complete mixes on the road in headphones. 
On center of the sphere - I placed it as I really sit in the room - 33% percent of front wall. I mean you've done a perfect work on reverberation realism. When I solo the left channel things just going right - I hear mid to high spectrum on the left channel with real world diffusion and the low end something between the center of room to my right channel.

So I think I solved an issue with phasing effects at the center. Just added Goodhertz Midside plugin and rise the mid channel by 4db and it's just gone and I can hear voices, snares and etc center information so clearly. So now I'm bypassing that correction with Soundobject impulse responce and comparing it with clear headphones (just cheap Superlux 681 but I love transient response on them) and realized that I never heard left channel properly due to amount of perceived high end of the right channel (and I believe it's not an issue of the headphones but just how my HRTF works). That's the massive difference! Because I never heard not only the true dynamics of the tracks but also instruments' decay and their position in sound field. Amazing experience with that, thanks again for your work!
